src/main/java/domain/ChessBoard.java

```diff
@@ -0,0 +1,42 @@
+package domain;
+
+import domain.piece.Piece;
+import domain.piece.Pieces;
+import domain.piece.attribute.point.Point;
+import dto.RouteDto;
+import factory.ChessBoardGenerator;
+
+import java.util.Map;
+
+public class ChessBoard {
+    private final Pieces pieces;
+
+    public ChessBoard(final Pieces pieces) {
+        this.pieces = pieces;
+    }
+
+    public Piece findPieceByPoint(final Point point) {
+        final var piece = this.pieces.findPieceWithPoint(point);
+        return piece.orElseThrow(() -> new IllegalArgumentException("해당 포인트에는 기물이 없습니다"));
```

Comment:
var을 쓸때와 안쓸때의 차이가 있나요? 하나로 통일하는게 좋아보입니다~
```suggestion
        return pieces.findPieceWithPoint(point)
            .orElseThrow(() -> new IllegalArgumentException("해당 포인트에는 기물이 없습니다"));
```

Reply:
var를 쓸때와 안쓸때의 차이는 없습니다. 시간이 타이트하고 페어 프로그램을 진행하다 보니 각자 스타일대로 혼용하여 사용한 곳 많은 것 같습니다.  수정하겠습니다 😊